loader/src/main/java/net/neoforged/fml/loading/moddiscovery/ModFileParser.java

```diff
@@ -85,17 +88,32 @@ protected static List<CoreModFile> getCoreMods(final ModFile modFile) {
                 .toList();
     }
 
+    private record PotentialMixin(String name, boolean optional) {}
```

Comment:
Suggestion: `PotentialMixinConfig` since it's a mixin config we're specifying, not an individual mixin

Reply:
Right, let me fix that.